src/vs/workbench/browser/layout.ts

```diff
@@ -172,6 +172,8 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 		protected readonly parent: HTMLElement
 	) {
 		super();
+
+		this._container.setAttribute('allowfullscreen', '');
```

Comment:
@rebornix should we only set this attribute when running in the web? Also, I found no mention that this attribute can be used outside of an `iframe`? Is there some docs?

Reply:
It turns out that `allowfullscreen` is not necessary. I read something on MDN talking about the element should be marked as `allowfullscreen` but I can't find it anymore and it seems wrong, as without that flag, it works.